### ISSUE TITLE
Suggest a cool logo for reejs

### ISSUE THREAD

**Issue by renhiyama** [rovelstars/reejs#11]
It should be:
- made by yourself
- be cool/be ree-tastic!
- not copyrighted by anyone else (follows point one)
- you agree to let us use it for branding, and others may use it in commercial purposes, as long as its in fair use (such as reejs tutorial on blogs & YouTube thumbnails, where it makes sense. One should not be able to resell the logo itself. While this code is MIT licensed, your/our logo wouldn't be.)
- readable on both light and dark background. Prefer a square sized logo.
- using the gradient that reejs currently uses on the website "Reejs" heading is preferred and recommended, if possible.

Let me know if anyone makes it!
Important note: its upto me to decide whether I like your logo or not. It could be a mascot, "angry pepe frog ree-ing" works too!

**Comment by mohitparmar1** [comment]
hey @renhiyama  i can make logo for Reejs!

**Comment by renhiyama** [comment]
@mohitparmar1 hi there, thanks for your interest! Reejs logo used `bg-gradient-to-r from-purple-600 to-pink-600` tailwind classes to achieve the gradient on its text, so here are the colors from tailwind website respectively:
- purple-600: `#9333ea`
- pink-600: `#db2777`

Let me know about your progress!

**Comment by mohitparmar1** [comment]
sir, I have to create a logo right? 
![Frame 3](https://github.com/rovelstars/reejs/assets/100152343/f408fcd7-20db-49dd-bcff-a917ab5c81d8)

**Comment by renhiyama** [comment]
> sir, I have to create a logo right? ![Frame 3](https://user-images.githubusercontent.com/100152343/272792411-f408fcd7-20db-49dd-bcff-a917ab5c81d8.png)

I wanted a logo, in the sense of something like diagram/illustration instead of text. If I was talking about text, the current gradient text already is good enough
![image](https://github.com/rovelstars/reejs/assets/92968534/487de259-c056-4ef7-9aac-37c905e1728a)

**Comment by vinayakhegde1** [comment]
@renhiyama what if add pepe emote instead of ree i dont know whether it will have any copyright issue also need to try first it can be achieved or not

**Comment by renhiyama** [comment]
I believe that best represents it.
pepe ree emoji.
Please consider getting a discord-like high res image. If you find a gif of the same size, also include it.

**Comment by renhiyama** [comment]
post them here .

**Comment by vinayakhegde1** [comment]
@renhiyama okay I'll paste if I get a good one

**Comment by vinayakhegde1** [comment]
@renhiyama bro can you tell me how do i run ree locally i dont find any steps if you could help

**Comment by renhiyama** [comment]
> @renhiyama bro can you tell me how do i run ree locally i dont find any steps if you could help 

Not sure how does that help in making the logo but here you go:
- for running the website, go into the website folder, run `npm i && npm run docs:dev`

- for running reejs cli locally, make sure you dont have reejs installed already, then run `npm i && npm link .` to link the local reejs code.

**Comment by vinayakhegde1** [comment]
@renhiyama i will try other issues as well

**Comment by vinayakhegde1** [comment]
@renhiyama hey what say we generate pepe ree logo from Midjourney with the same color scheme would it be legal?

**Comment by renhiyama** [comment]
Not sure. I guess I need to create it myself then.

**Comment by vinayakhegde1** [comment]
@renhiyama yes but copyright is the only issue or else i think pepe would be great logo

**Comment by pravintargaryen** [comment]
@renhiyama Is it still open. Have you created the logo yet. Can i give it a try?

**Comment by renhiyama** [comment]
Hi @pravintargaryen , feel free to create an abstract logo, similar to vite's. Use the gradient mentioned above and if possible, try to give some glow effect and stuff.

**Comment by ArnavK-09** [comment]
![_4c9d91b3-8a4c-45ff-ad59-6fcb34bd1c79_adobe_express (1).png](https://github.com/rovelstars/reejs/assets/69188140/9d91d76d-ae92-4499-a222-10faf2c4a314)

**Comment by ArnavK-09** [comment]
![_fb562ada-cf09-4107-a541-ad1581304795_adobe_express (1).png](https://github.com/rovelstars/reejs/assets/69188140/dafe0825-b978-473b-b346-47a38efcde0e)

**Comment by ArnavK-09** [comment]
![_08dcd58d-a57c-4676-bb91-9aed3fe03beb_adobe_express (1).png](https://github.com/rovelstars/reejs/assets/69188140/5710a57a-093b-446b-806c-afc5ed1193fc)

**Comment by ArnavK-09** [comment]
![_84c68f6e-2f47-42e8-8208-e513278ae391_adobe_express (1).png](https://github.com/rovelstars/reejs/assets/69188140/b0f79fc9-e3d9-4ef8-9918-cd2ce6c11704)

**Comment by renhiyama** [comment]
@ArnavK-09 all of these logos feel too complicated, they have too much going on, its AI generated for sure, two of them are plain "R" letter with designs around it, one is a pixelated guy (resembles nothing with reejs) and one is a bot (resembles nothing again ...)

**Comment by ArnavK-09** [comment]
> @ArnavK-09 all of these logos feel too complicated, they have too much going on, its AI generated for sure, two of them are plain "R" letter with designs around it, one is a pixelated guy (resembles nothing with reejs) and one is a bot (resembles nothing again ...)

乁⁠(⁠ ⁠.⁠ ⁠ര⁠ ⁠ʖ̯⁠ ⁠ര⁠ ⁠.⁠ ⁠)⁠ㄏ
Just sent some inspirations
Yes , gen

**Comment by Gitnikkx** [comment]
Hey👋🏻 @renhiyama  In case you haven't found a logo yet, I’ve added one. I also noticed there’s no favicon on the website, so I’ve attached one separately for you to use !!
![reejs](https://github.com/user-attachments/assets/146e1fd0-0d4e-493e-9425-8df6829a0c94)
![reejs-favicon](https://github.com/user-attachments/assets/bd392c42-dafb-4917-8ace-6b13ad3424f1)
